<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>First number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>121</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Second number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="etFirstNum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>113</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="etSecondNum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>113</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btPlus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btMinus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btDiv">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>\</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btMult">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Result is:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="etResult">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>113</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
